### DISCUSSION TITLE
关于ELECTRONBOT 组装禁锢连接件问题

### DISCUSSION THREAD

**Discussion by wenFrankieeee** [peng-zhihui/ElectronBot#147] [General]
如题，请问各位在 组装过 电路板到模型 以及 各模型间 连接 及禁锢都 采用什么型号的螺丝？ 长度及半径？ 谢谢

**Comment by txp666** [comment]
建议买个螺丝大礼包，我用的m2*4、5、6自攻螺丝
![7675045f4aa48964fc9104275cb38e3](https://user-images.githubusercontent.com/52451470/162730808-a328d854-63fa-48b4-ae8b-df527a4571d8.jpg)

**Reply by wenFrankieeee** [reply]
谢谢，能发个某宝链接吗？
